.github/actions/php-pipeline/phpunit.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+./includes.sh
+export CMD=phpunit
+
+function run_packages_tests {
+	export CMD_PACKAGES="composer phpunit"
+	export PACKAGES='./packages/**/tests/php'
+	for PACKAGE in $PACKAGES
+	do
+		if [ -d "$PACKAGE" ]; then
+			cd "$PACKAGE/../.."
+			export NAME=$(basename $(pwd))
+
+			if [ ! -e tests/php/travis-can-run.sh ] || tests/php/travis-can-run.sh; then
```

Comment:
We should probably rename that `travis-can-run.sh` file.

Or since we're running the tests as `composer phpunit` anyway, maybe we should put the check in the package's composer.json instead of having the test runner do it. That was kind of a bad idea on my part.

Reply:
I moved PHP version check into a bootrsrap file